### PR TITLE
Initialize compute nodes once in TunnellingWithAgent tests

### PR DESCRIPTION
**Pull request by mzaremba04**
This will better reflect normal user environment and hopefully reduce
random tests' failures.

### CODE REVIEW

**Comment by mzaremba04** [comment]
recheck no bug

File: Test/PesterLogger/RemoteLogCollector.ps1 [hunk 1]
```diff
@@ -187,3 +187,10 @@ function Merge-Logs {
 
     Write-Log ("=" * 100)
 }
+
+function Clear-Logs {
+    Param([Parameter(Mandatory = $true)] [LogSource[]] $LogSources)
+    foreach ($LogSource in $LogSources) {
+        $LogSource.ClearContent()
+    }
+}
```
Comment (sodar) [inline Test/PesterLogger/RemoteLogCollector.ps1]: Please add a new line.

File: Test/PesterLogger/RemoteLogCollector.ps1 [hunk 2]
```diff
@@ -44,7 +44,7 @@ class FileLogSource : LogSource {
                 }
             } else {
                 foreach ($File in $Files) {
-                    $Content = Get-Content -Raw $File
+                    $Content = Get-Content -Force -Raw $File
```
Comment (sodar) [inline Test/PesterLogger/RemoteLogCollector.ps1]: Since `Out-File -Append` does not do a read lock on a file, we can remove `-Force` flag.

**Comment by mzaremba04** [comment]
recheck no bug

**Comment by sodar** [comment]
recheck no bug

**Comment by aethyrxeo** [comment]
recheck no bug

**Pull request merged**